1943 feat action palette (#1955)
pluralsight/pando: docs/src/content/docs/explanation/color.md

A palette that represents actionable and interactive items (eg. buttons, text links). Only used for status and feedback.

> <a href='/reference/colors/action-palette'>view more about action colors</a>

Reviewer: There were a few articles back in the day mentioning that "view more" is not accessible in links.

```suggestion
<a href='/reference/colors/action-palette'>View the action color reference docs</a>
```
